arbutus/theme.json

```diff
@@ -47,66 +47,28 @@
 			"fontFamilies": [
 				{
 					"fontFamily": "\"Roboto\", sans-serif",
-					"slug": "roboto",
-					"name": "roboto",
-					"google": "family=Open+Sans:ital,wght@0,300;0,400;0,500;0,700;0,900;1,300;1,400;1,500;1,700;1,900"
+					"fontSlug": "roboto",
+					"slug": "body-font",
+					"name": "Body (Roboto)",
+					"google": "family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,800;1,900"
```

Comment:
what happens to child themes that don't make these changes? will this update break their sites?

Reply:
If they update Blockbase and not their child then it _may_ break their sites depending on what the child theme.json file looks like. In most cases the font declarations at present are more specific than this proposal, so they should continue working the same.